presto-parquet/src/main/java/io/prestosql/parquet/predicate/PredicateUtils.java

```diff
@@ -161,20 +162,23 @@ private static boolean isColumnPredicate(ColumnDescriptor columnDescriptor, Tupl
 
     @VisibleForTesting
     @SuppressWarnings("deprecation")
-    public static boolean isOnlyDictionaryEncodingPages(Set<Encoding> encodings)
+    static boolean isOnlyDictionaryEncodingPages(ColumnChunkMetaData columnMetaData)
     {
-        // TODO: update to use EncodingStats in ColumnChunkMetaData when available
+        // Newer versions will have EncodingStats available
```

Comment:
change comment to: Parquet V2 has EncodingStats

Reply:
I put "newer" here mainly because if I use parquet-mr 1.9.0 (which knows about `EncodingStats`) but configure the writer to use the v1 format, it will still write out the page encoding stats. So, I guess technically it's more of a construct of newer versions of the library than necessarily the v2 format (but this was introduced originally with the v2 format). I'll switch the comment out.

Reply:
sure, then put parquet-mr 1.9.0 there?